ci/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+USER_ID=${LOCAL_UID:-9001}
+GROUP_ID=${LOCAL_GID:-9001}
+USER_NAME=${LOCAL_USER:-user}
+WORKING_DIR=${BASE_ROOT_DIR}
+
+echo "Starting with UID: $USER_ID, GID: $GROUP_ID"
+useradd -u $USER_ID -o -m $USER_NAME
+groupmod -g $GROUP_ID $USER_NAME
+
+chown -R $USER_NAME:$USER_NAME ${WORKING_DIR}
+tail -f /dev/null
```

Comment:
what is this line doing and why?

Reply:
Without this line, `docker run ..` would hit the entrypoint.sh script and then exit once the script was finished. Adding this line keeps the container running without using any CPU inside the container so that we can then run all of the `docker exec` commands against the running container

Reply:
Why not `sleep inf`?

Reply:
I don't have a strong opinion about one over the other. I've almost always seen `tail -f /dev/null` as the recommended solution for docker and IIRC `sleep inf` had some issues with portability back in the day